### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by AdrianValdes**

### CODE REVIEW

File: adrian/script.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+console.log(Math.max(a, b, c));
+// Create a new string adding "Py" in front of a given string. If the given string begins with "Py" then print the original string.
+let givenString = "hello";
+let newString = `Py${givenString}`;
+let originalString = "The original string?";
+if (givenString.slice(0, 2) === "Py") console.log(originalString);
+// Calculate the sum of the two integers. If the sum is in the range 50 to 80, print 65, otherwise print 80.
+let integer3 = 34;
+let integer4 = 17;
+if (integer3 + integer4 > 50 && integer3 + integer4 < 70) console.log(65);
```
Comment (ciamac-da) [inline adrian/script.js]: In this condition the range is between 50.000000000001 and 69.9999999 and not between 50 and 80

File: adrian/script.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+let a = 0;
+let b = 1;
+let c = 2;
+//c = 34
+console.log(Math.max(a, b, c));
+// Create a new string adding "Py" in front of a given string. If the given string begins with "Py" then print the original string.
+let givenString = "hello";
+let newString = `Py${givenString}`;
+let originalString = "The original string?";
+if (givenString.slice(0, 2) === "Py") console.log(originalString);
```
Comment (ciamac-da) [inline adrian/script.js:29]: Here you cand also add else to get Pyhello

**Review by ciamac-da — CHANGES_REQUESTED** [review]

File: adrian/script.js (hunk 2 above)
Comment (AdrianValdes) [reply adrian/script.js:29]: Done

File: adrian/script.js (hunk 1 above)
Comment (AdrianValdes) [reply adrian/script.js]: Done, I did it now to include 50 and 80

**Review by ciamac-da — APPROVED** [review]
👍🏻

**Pull request merged**